main/guides/getting-started/index.md

```diff
@@ -39,20 +39,49 @@ In this section you'll be installing prerequisite components into your environme
 ### Installing Node.js
 
 <details>
-<summary>Installing Node.js on Ubuntu</summary>
+<summary>Installing Node.js</summary>
 
 This tutorial requires a specific version of [Node.js](https://nodejs.org/).
 
 The [nvm](https://github.com/nvm-sh/nvm) (Node Version Manager) utility makes it easy to select the right version.
 
 First, follow the [nvm installation instructions](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating), including sourcing `nvm.sh` into the shell.
+
+<details>
+<summary>Installing nvm on Ubuntu</summary>
+
+You can copy the following command on your shell and get it running
+
 ```sh
 # For example:
 wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
 export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
 ```
 
+</details>
+
+<details>
+<summary>Installing nvm on MacOS</summary>
```

Comment:
Do you plan to add ci for MacOS?

Reply:
Apologies for this.  Came from another branch. Fixed now